draft-ietf-ccwg-rfc5033bis.md

```diff
@@ -326,9 +327,9 @@ the algorithm reduces its sending rate is algorithm specific, but see
 Bufferbloat {{Bufferbloat}} refers to the building of long queues in
 the network. Many network routers are configured with very large buffers.
 If congestion starts happening, classic TCP congestion control algorithms
-{{!RFC5681}} will continue sending at a high rate until the buffer fills
-up completely and packet losses occur. Every connection going through
-that bottleneck will experience high latency.  This adds unwanted latency that
+{{!RFC5681}} will continue sending at a high rate until a FIFO buffer fills
```

Comment:
IIUC this will be the first reference to FIFO, so please expand the acronym and delete the expansion later in the document.